packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -76,6 +74,12 @@ make smaller updates to your app.
 
   @override
   Future<int> run() async {
+    logger.warn(
+      '''
+"shorebird release" is deprecated.
```

Comment:
```suggestion
"shorebird release" is deprecated and will be removed in a future release.
```

Reply:
I intentionally didn't do that because the command "shorebird release" technically won't be removed 🤣

Reply:
Hahaha fair enough!